docs/install/_advanced.rst

```diff
@@ -8,44 +8,21 @@ Advanced installation options
 Generic installation (pip install)
 ----------------------------------
 
-Once Kolibri is released, you may install it as a standard package from PyPi using this command::
-
-    pip install kolibri --pre
-
-
-.. _ppa-installation:
-
-Debian/Ubuntu: Subscribe to updates through a PPA
--------------------------------------------------
-
-We maintain a `PPA on Launchpad <https://launchpad.net/~learningequality/+archive/ubuntu/kolibri>`__ and if you are connected to the internet, this will also give you automatic updates.
-
-On Ubuntu/Debian/Raspbian, do this::
-
-    sudo apt-get install software-properties-common python-software-properties
-    sudo add-apt-repository ppa:learningequality/kolibri
-    sudo apt-get update
-    sudo apt-get install kolibri
+You can install Kolibri as a standard package from PyPi (works on Mac, Windows, and Linux) using this command::
 
+    pip install kolibri
 
 .. _raspberry-pi-wifi:
 
 Raspberry Pi
 ------------
 
 .. note::
-  This section is pending instructions for creating a WI-FI hotspot. You can
-  install Kolibri on Raspbian following the above instructions for the PPA or
-  the ``pip install`` method.
+  This section is pending instructions for creating a WI-FI hotspot on Raspberry Pi. Follow the instructions above to install Kolibri on :ref:`Raspbian <rpi>`.
 
 
 Nginx configuration
 -------------------
 
 .. note::
-  This section is pending instructions for NGINX configuration. You can
-  install Kolibri on Raspbian following the above instructions for the PPA or
-  the ``pip install`` method. If you are an experienced system administrator or
-  Django user, know that it is possible to use Kolibri with for instance
-  UWSGI+Nginx, using conventional methods.
-
+  This section is pending instructions for NGINX configuration. You can install Kolibri on Raspbian following the above instructions for the PPA or the ``pip install`` method. If you are an experienced system administrator or Django user, know that it is possible to use Kolibri with for instance UWSGI+Nginx, using conventional methods.
```

Comment:
A few things still bother me about this section...

* it's not clear to me what Raspbian has to do with Ngninx?
* We should standardize on one capitalization, probably 'Nginx'
* having stuff like "This section is pending" to seems unhelpful and just adds noise

-----

![image](https://user-images.githubusercontent.com/2367265/38647964-c0fab848-3da3-11e8-8492-2358d320606b.png)

I'd recommend removing the entire 'advanced' section, and replacing it all with a single top-level "pip install" section.